index.js

```diff
@@ -34,15 +34,21 @@ Microcomponent.prototype.on = function (eventname, handler) {
   assert.equal(typeof handler, 'function', 'microcomponent.on handler should be type function')
 
   if (eventname === 'render') {
-    this._render = function (props) {
+    this._render = function () {
```

Comment:
Oh, re-read your comments. This is err, woah quite strange haha. Ah well, we can always clean it up later